app/src/main/java/org/sopt/do_sopt_compose/ui/pages/Loginpage.kt

```diff
@@ -0,0 +1,101 @@
+package org.sopt.do_sopt_compose.ui.pages
+
+import android.util.Log
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.material3.Scaffold
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.lifecycle.viewmodel.compose.viewModel
+import org.sopt.do_sopt_compose.R
+import org.sopt.do_sopt_compose.ui.UiStatus
+import org.sopt.do_sopt_compose.ui.components.MainButton
+import org.sopt.do_sopt_compose.ui.components.PasswordTextField
+import org.sopt.do_sopt_compose.ui.components.TextField
+import org.sopt.do_sopt_compose.ui.components.TitleText
+import org.sopt.do_sopt_compose.ui.pages.states.LoginPageState
+import org.sopt.do_sopt_compose.ui.pages.viewmodels.LoginViewModel
+
+@Composable
+fun LoginPage(
+    state: LoginPageState,
+    onNavigateToSignUp: () -> Unit,
+    onNavigateToMainUp: () -> Unit,
+) {
+    val loginViewModel: LoginViewModel = viewModel()
+
+    Scaffold(
+        topBar = { TitleText(text = "login") },
+        content = {
```

Comment:
이러한 content도 외부로 빼면 좋을 것 같습니다.!
```suggestion
    Scaffold(
        topBar = { TitleText(text = "login") },
        ) {
               ...
               }
```

Reply:
그리고 topbar에 저는 TopBar를 넣었는데 Text를 넣는거랑 차이가 뭔지 찾아봐야겠군요.!